RELEASE_NOTES.md

```diff
@@ -16,27 +16,55 @@
 
 * **IMPORTANT** Systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
 
+### New software support
+
+* Bump `iuf-cli` version to 1.6.15
+* Bump `cray-nls` version to 4.0.15
+* Added support to upgrade CSM through IUF
+* Added support for iSCSI based boot content projection for `rootfs` and `PE` images
+
 ### Automation improvements
 
 ### Base platform component upgrades
 
 | Platform Component           | Version        |
 |------------------------------|----------------|
+| `Kubernetes`                 | 1.24.17        |
+| `istio`                      | 1.19.10        |
+| `kiali`                      | 1.75.0         |
+| `Strimzi Kafka`              | 0.41.0         |
+| `Kyverno`                    | 1.10.7         |
 
 ### Security improvements
 
+* `Kyverno` is upgraded from 1.9.5 version to 1.10.7 version to address CVEs.
+
 ### Customer-requested enhancements
 
+* iSCSI based boot content projection for `rootfs` and `PE` images.
+
 ### Documentation enhancements
 
```

Comment:
Can we also add the following under line 46 for "documentation enhancements"?

* The SAT documentation moved to be fully included within the [System Admin Toolkit (SAT) section](https://cray-hpe.github.io/docs-csm/en-16/operations/system_admin_toolkit/) of the [CSM Administration Guide](https://cray-hpe.github.io/docs-csm/en-16/operations/).

Reply:
Doe